curriculum/challenges/english/10-coding-interview-prep/rosetta-code/discordian-date.md

```diff
@@ -10,12 +10,24 @@ dashedName: discordian-date
 
 The Gregorian calendar is a solar calendar with 12 months of 28–31 days each. The year in both calendars consists of 365 days, with a leap day added to February in the leap years except for three centurial years every 400 years.
 
-The Discordian calendar is aligned with the Gregorian calendar and begins on January 1. Its week consists of five days, there are 73 weeks per year. Every fourth year in the Discordian calendar, an extra day is inserted between Chaos 59 and Chaos 60
+The <a href="http://www.rosettacode.org/wiki/Discordian_date" target="_blank" rel="noopener noreferrer nofollow">Discordian calendar</a> is aligned with the Gregorian calendar and begins on January 1. Its week consists of five days, there are 73 weeks per year. Every fourth year in the Discordian calendar, an extra day is inserted between Chaos 59 and Chaos 60.
 
-Note that in the Discordian calendar Chaos 1, 3188 YOLD is January 1, 2022, in the Gregorian calendar.
+here are the structure of Discordian calendar.
```

Comment:
```suggestion
The months, weekdays, and events in the Discordian calendar are given by:
```

Apostle and Holidays are not mentioned above, so there is no context for that data